src/setup/rgw.py

```diff
@@ -178,6 +178,20 @@ def init(conf: MappedConf):
             Conf.delete(rgw_consul_idx, rgw_lock_key)
             Log.info(f'{rgw_lock_key} key is deleted')
 
+        Log.info('Starting radosgw service.')
+        try:
+            os.system("sh /opt/seagate/cortx/rgw/bin/rgw_service")
```

Comment:
As discussed, keep a TODO to replace os.system with SimpleProcess calls. i know yday we tried and ran into issues, but we should not miss this, as it has security concerns

Reply:
Ack